portfolio/src/main/webapp/script.js

```diff
@@ -33,17 +33,36 @@ function addRandomFact() {
  * combines all of the above code into a single Promise chain. You can use
  * whichever syntax makes the most sense to you.
  */
-function getRandomQuoteUsingArrowFunctions() {
-  fetch('/data').then(response => response.text()).then((comment) => {
-    document.getElementById('comment-container').innerText = comment;
-  });
+function getTranslation() {
+    const lang = document.getElementById('language').value;
+    console.log(lang);
```

Comment:
I think we can remove this console.log() too since we don't need debugging output in production.